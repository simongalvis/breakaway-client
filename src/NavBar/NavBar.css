
h3{
    float:left;

}
nav{
   
}
header{
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 5%;
    background-color:white;
    padding: 0px 16px;
    border-bottom: 1px solid rgb(231, 231, 231);
    min-height: 55px;
    
}
img {
    height:40px;
    width: 105px
}
button{
    height:30px;
    
    border: 1px transparent solid;
    border-color: rgb(231, 231, 231);
    font-weight: 700;
}
#login-button{
    background-color: white;
    border-radius:4px;
    padding: 1px 7px;
    height:31px;
    width:74px;
}