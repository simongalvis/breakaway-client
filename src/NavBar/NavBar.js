import React from 'react';
import { Link, BrowserRouter} from 'react-router-dom';
import './NavBar.css';
import logo from '../images/breakaway-logo.png'



class NavBar extends React.Component{



    render(){
        return(
            
                
                <header>
                <Link to='/' >
                    <img src={logo} alt='Blue breakaway logo' id='main-logo'/>
                </Link>
                <Link to="/login-page" >
                    <button className='nav-button' id='login-button' value='login'>Log In</button>
                </Link>
                </header>
                
               
                
                
            
            
        )
    }
}


export default NavBar;