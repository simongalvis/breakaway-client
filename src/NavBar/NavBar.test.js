import React from 'react'
import ReactDOM from 'react-dom'
import NavBar from './NavBar'
import renderer from 'react-test-renderer';

describe(`NavBar component`, () =>{
  it(`renders without crashing`, () =>{
    const div = document.createElement('div');
  
    ReactDOM.render(<NavBar/>, div);
  
    ReactDOM.unmountComponentAtNode(div);
  })
  it(`Renders UI as expected`, () =>{
    const tree = renderer
      .create(<NavBar/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})
