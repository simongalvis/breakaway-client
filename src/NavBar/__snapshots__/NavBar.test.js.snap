// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component Renders UI as expected 1`] = `
<header>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Blue breakaway logo"
      id="main-logo"
      src="breakaway-logo.png"
    />
  </a>
  <a
    href="/login-page"
    onClick={[Function]}
  >
    <button
      className="nav-button"
      id="login-button"
      value="login"
    >
      Log In
    </button>
  </a>
</header>
`;
